exoservice-javascript/bin/spec

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
+webpack
```

Comment:
Let's follow our convertion for helper scripts:
- `build` compiles source code into executable formats, for example via `lsc` or `webpack`
- `spec` calls `build`